### ISSUE TITLE
Column filter of References panel doesn't show a result for some criteria

### ISSUE THREAD

**Issue by kseniiaguzeeva** [dbeaver/dbeaver#6372]
#### System information: 
- Operating system (distribution) and version Windows 10
- DBeaver version 6.1.4
- Java version 11
- Additional extensions

#### Connection specification:
- Database name and version MySQL 8+
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
Column filter of References panel doesn't show a result for some criteria
#### Steps to reproduce, if exist:
1. open MySQL 8+ db> sakila>address> Data tab
2. select a whole column of the table and open References panel
3. Check that Reference is "city (fk_address_city)"> open "filter by column value" in column "City"> select "Abu Dhabi" in the opened list.

Expected result: raw is shown with "Abu Dhabi" value in column "city" of the reference table
Actual result: No raw is shown in the Reference table

Note: the raw with "Abu Dhabi" value in column "city" will be shown if to sort column "city_id" ASC in the source table and select the whole column again after step 2. 

#### Include any warning/errors/backtraces from the logs

![6311 (2)](https://user-images.githubusercontent.com/51405061/62053978-5b7b8f80-b221-11e9-8244-5d3d64611e30.gif)

**Comment by serge-rider** [comment]
It is not a bug, more like a misfunction.

When we show possible filter values we query for possible values of some column (city name) in referenced table (city). "Abu Dhabi" is valid city name. But there  are no selected addresses with this city.
So yes, ref panel shows all possible value and doesn't take into original table values. It will be quite complicated to implement (and will require series of sub-selects which are not supported by generic SQL engine).
So - won't fix, at least for now.
